Assignment 5.Rmd

```diff
@@ -15,20 +15,32 @@ library(party)
 ```
 
 ## Part I
+
+Code Book
+
+id - student id prior_prob_count - The number of problems a student has done in the system prior to the surrent session
+score - The score the student achieved in the current session
+hints - The number of hints the student requested in the current session
+hint.y - Whether or not the student asked for hints in the current session
+complete - Whether or not the student completed the current session
+action - The action suggested by the system to a teacher about a given student based on their performance
+
 ```{r}
-D1 <- 
+D1 <- read.csv("intelligent_tutor.csv")
 ```
 
 ##Classification Tree
 First we will build a classification tree to predict which students ask a teacher for help, which start a new session, or which give up, based on whether or not the student completed a session (D1$complete) and whether or not they asked for hints (D1$hint.y). 
 ```{r}
-
-c.tree <- rpart(action ~ hint.y + complete, method="class", data=D1) #Notice the standard R notion for a formula X ~ Y
+?rpart
+c.tree <- rpart(action ~ hint.y + complete, method="class", data=D1) 
+#Notice the standard R notion for a formula X ~ Y
 
 #Look at the error of this tree
 printcp(c.tree)
 
 #Plot the tree
+#?
```

Comment:
You have a question here? The plot creates a `postscript` file (like a pdf) that should appear in your project directory.